src/tests/run-block-actions.test.ts

```diff
@@ -1,5 +1,6 @@
 import { assertEquals, assertExists, assertRejects } from "../dev_deps.ts";
 import { RunBlockAction } from "../run-block-actions.ts";
+import { extractBaseHandlerArgsFromPayload } from "../dispatch-payload.ts";
```

Comment:
I don't like that we have to pull in an auxiliary function from `dispatch-payload.ts` in this unit test file focussed on executing `run-block-actions.ts` code.
Perhaps moving that function to a utility file/module helps? Not sure. Let me know if anyone has thoughts.